modules/swagger-parser-v3/src/test/java/io/swagger/parser/test/OpenAPIParserV3Test.java

```diff
@@ -0,0 +1,141 @@
+package io.swagger.parser.test;
+
```

Comment:
Be sure to add some tests that pass `auth` and `ParseOptions` because those are needed for proper resolving